spec/unit/config_spec.rb

```diff
@@ -21,6 +21,18 @@
   include FakeFS::SpecHelpers
   subject { Machinery::Config.new }
 
+  it "works with keys with '-'s" do
```

Comment:
I had to read it twice to get it. How about:
`it "works with keys containing hyphens ('-')" do`